src/components/forms/formr-part-b/View.tsx

```diff
@@ -76,73 +80,143 @@ const View = ({ canEdit, history }: IView) => {
-              <SummaryList.Value>{formData.dualSpecialty}</SummaryList.Value>
-            </SummaryList.Row>
-          </SummaryList>
-        </Panel>
-
+              <SummaryList.Row>
+                <SummaryList.Key>Dual Specialty</SummaryList.Key>
+                <SummaryList.Value>{formData.dualSpecialty}</SummaryList.Value>
+              </SummaryList.Row>
+            </SummaryList>{" "}
+            <SummaryList>
+              <SummaryList.Row>
+                <SummaryList.Key>Forename</SummaryList.Key>
+                <SummaryList.Value>{formData.forename}</SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
+                <SummaryList.Key>GMC-Registered Surname</SummaryList.Key>
+                <SummaryList.Value>{formData.surname}</SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
+                <SummaryList.Key>GMC Number</SummaryList.Key>
+                <SummaryList.Value data-cy="gmcNumber">
+                  {formData.gmcNumber}
+                </SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
+                <SummaryList.Key>Email Address</SummaryList.Key>
+                <SummaryList.Value>{formData.email}</SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
+                <SummaryList.Key>Deanery / HEE Local Team</SummaryList.Key>
+                <SummaryList.Value data-cy="localOfficeName">
+                  {formData.localOfficeName}
+                </SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
+                <SummaryList.Key>
+                  Previous Designated Revalidation Body
+                </SummaryList.Key>
+                <SummaryList.Value>{formData.prevRevalBody}</SummaryList.Value>
+              </SummaryList.Row>
+              {formData.prevRevalBodyOther && (
+                <SummaryList.Row>
+                  <SummaryList.Key>
+                    Other Previous Revalidation Body
+                  </SummaryList.Key>
+                  <SummaryList.Value>
+                    {formData.prevRevalBodyOther}
+                  </SummaryList.Value>
+                </SummaryList.Row>
+              )}
+              <SummaryList.Row>
+                <SummaryList.Key>Current Revalidation Date</SummaryList.Key>
+                <SummaryList.Value data-jest="currRevalDate">
+                  {DateUtilities.ToLocalDate(formData.currRevalDate || null)}
+                </SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
+                <SummaryList.Key>Date of Previous Revalidation</SummaryList.Key>
+                <SummaryList.Value data-jest="prevRevalDate">
+                  {DateUtilities.ToLocalDate(formData.prevRevalDate || null)}
+                </SummaryList.Value>
+              </SummaryList.Row>
+              <SummaryList.Row>
```

Comment:
I couldn't see where this came from, new content?

Reply:
![image](https://user-images.githubusercontent.com/42062738/181746126-6bae176d-2eab-4444-81d0-5b83a797926c.png)
No new content just not very user friendly. There is a ticket to refactor the View comp. Just needs refining to agree the best way before diving in.

Reply:
Ah, I see.
I turned whitespace comparison off and it flagged it all as new code without showing the old 🤦🏻